2024/migrant-issue-shelters-bar/main.js

```diff
@@ -0,0 +1,166 @@
+            1499.0, 1508.0, 1515.0, 1578.0, 1664.0, 2027.0, 2202.0, 2203.0,
+            2272.0, 2671.0, 3145.0, 3238.0, 3549.0, 3561.0, 3537.0, 3634.0,
+            3749.0, 4280.0, 4541.0, 4609.0, 4805.0, 4965.0, 4989.0, 4993.0,
+            4876.0, 4628.0, 4564.0, 4346.0, 4267.0, 4121.0, 3992.0, 3740.0,
+            3451.0
+        ]
+    },
+    {
+        name: 'West',
+        data: [
+            287.0, 287.0, 200.0, 197.0, 200.0, 192.0, 196.0, 198.0, 198.0,
+            198.0, 195.0, 200.0, 200.0, 197.0, 200.0, 881.0, 1578.0, 1835.0,
+            2205.0, 2247.0, 2419.0, 2408.0, 2828.0, 2850.0, 2890.0, 3244.0,
+            3466.0, 3788.0, 3933.0, 3935.0, 3915.0, 4125.0, 4079.0, 4004.0,
+            3920.0, 3772.0, 3687.0, 3552.0, 3506.0, 3352.0, 3296.0
+        ]
+    }
+];
+
+shelterCategories = [
+    '06-09-2023',
+    '06-16-2023',
+    '06-23-2023',
+    '06-30-2023',
+    '07-07-2023',
+    '07-14-2023',
+    '07-21-2023',
+    '07-28-2023',
+    '08-04-2023',
+    '08-11-2023',
+    '08-18-2023',
+    '08-25-2023',
+    '09-01-2023',
+    '09-08-2023',
+    '09-15-2023',
+    '09-22-2023',
+    '09-29-2023',
+    '10-06-2023',
+    '10-13-2023',
+    '10-20-2023',
+    '10-27-2023',
+    '11-03-2023',
+    '11-10-2023',
+    '11-17-2023',
+    '11-24-2023',
+    '12-01-2023',
+    '12-08-2023',
+    '12-15-2023',
+    '12-22-2023',
+    '12-29-2023',
+    '01-05-2024',
+    '01-12-2024',
+    '01-19-2024',
+    '01-26-2024',
+    '02-02-2024',
+    '02-09-2024',
+    '02-16-2024',
+    '02-23-2024',
+    '03-01-2024',
+    '03-08-2024',
+    '03-15-2024'
+];
+
+Highcharts.setOptions({
+    lang: {
+        thousandsSep: ','
+    }
+});
+
+Highcharts.chart('chart-div', {
+    // Setting default colors
+    colors: [
+        '#800000',
+        '#FFA319',
+        '#C16622',
+        '#8F3931',
+        '#8A9045',
+        '#58593F',
+        '#155F83',
+        '#350E20',
+        '#47B5FF',
+        '#FF3399'
+    ],
+    chart: {
+        type: 'column'
+    },
+    title: {
+        text: 'Total Migrants at Shelters By Region From June 2023 to March 2024',
+        align: 'center'
+    },
+    subtitle: {
+        text: "Source: The 40th Ward Office's <a href=https://40thward.org/cirr/new-arrivals/new-arrivals-data-dashboard/>New Arrivals Data Dashboard</a> ",
+        align: 'center'
+    },
+    yAxis: {
+        min: 0,
+        title: {
+            text: ''
+        },
+        labels: {
+            formatter: function () {
+                return Highcharts.numberFormat(this.value, 0, '', ',');
+            }
+        },
+        stackLabels: {
+            enabled: false
+        }
+    },
+    legend: {
+        align: 'left',
+        x: 100,
+        verticalAlign: 'top',
+        y: 60,
+        floating: true,
+        backgroundColor:
+            Highcharts.defaultOptions.legend.backgroundColor || 'white',
+        borderColor: '#CCC',
+        borderWidth: 1,
```

Comment:
What's the default value for `borderWidth`?

Reply:
Defaults to 0